File: patch_imports.js
const fs = require('fs');
let code = fs.readFileSync('src/components/admin/AdminViews2.tsx', 'utf8');

// Ensure Clock and Trash are imported
let lines = code.split('\n');
let modified = false;

for (let i = 0; i < lines.length; i++) {
  if (lines[i].includes("from 'lucide-react'") && !lines[i].includes(" Trash")) {
     lines[i] = lines[i].replace("} from 'lucide-react'", ", Trash, Clock } from 'lucide-react'");
     modified = true;
     break;
  }
}

fs.writeFileSync('src/components/admin/AdminViews2.tsx', lines.join('\n'));
console.log("Imports patched");

File: patch_imports.py
with open('src/components/admin/AdminViews2.tsx', 'r') as f:
    content = f.read()

import re
# find lucide-react import and add Trash, Clock
match = re.search(r'import\s+\{(.*?)\}\s+from\s+[\'"]lucide-react[\'"]', content)
if match:
    imports = match.group(1)
    if 'Trash' not in imports:
        imports += ', Trash'
    if 'Clock' not in imports:
        imports += ', Clock'
    
    new_import = f"import {{{imports}}} from 'lucide-react'"
    content = content[:match.start()] + new_import + content[match.end():]

    with open('src/components/admin/AdminViews2.tsx', 'w') as f:
        f.write(content)
    print("Imports patched")
